src/shared/components/CallToActionButton/CallToActionButton.style.ts

```diff
@@ -37,7 +37,9 @@ export const ContentWrapper = styled.div`
   transition: all ${transitions.timings.regular} ${transitions.easing};
 `
 
-export const StyledContainer = styled.button<Omit<CallToActionButtonProps, 'label'>>`
+export const StyledContainer = styled('button', { shouldForwardProp: (prop) => prop !== 'colorVariant' })<
```

Comment:
I feel little guilty, but I didn't expect that. Now link CTA button doesn't do anything, so it should be this `shouldForwardProp` thing 😅 .  I looked in the chrome devtools and I see: 
![image](https://user-images.githubusercontent.com/51168865/127282076-b97320d1-c4ea-4c29-9441-be8da4f8c52e.png)
We should see proper` <a>` tag here, instead of button. Not sure how to fix this, though. If it's a problematic maybe we could use button and use `useNavigate` from `react-router-dom` in the end. But at least try to fix that using anchor tag

Reply:
Okay, I just figured it out. Opened PR on your branch. Let me know what you think.

Reply:
Oh, I missed that. Looks good!